src/helpers/questionMapping.ts

```diff
@@ -1,3 +1,15 @@
+import type { QuestionPropertyModel } from '../types/question_property.model';
+
+const SupportedQuestionTypes = {
+  yesno: true,
+  list_dropdown: true,
+  bootstrap_dropdown: true,
+  listradio: true,
+  numerical: true,
+  multipleshorttext: true,
+  multiplechoice: true
+} as const;
+
```

Comment:
https://github.com/ait-cs-IaaS/koord2ool/blob/main/src/helpers/questionMapping.ts

This should use this instead!

QuestionMapping should return either the charttype or None if not supported.